docs/databases/redis/deploy-redis-on-ubuntu-14-04.md

```diff
@@ -1,16 +1,16 @@
 ---
 author:
-  name: Sergey Pariev
-  email: spariev@gmail.com
+  name: Linode Community
+  email: docs@linode.com
 description: 'Deploy Redis on Ubuntu 14.04 LTS. This Tutorial Guides You Through Installation and Best Practices of Redis, an Open-Source, In-Line Memory Data-Structure Store.'
```

Comment:
Thanks for catching these, Sergey. I'm making a few adjustments and will send it back.
Re Line 90: while it was incorrect in the markdown, it was being rendered as though the numbers restarted because it was beyond the subheader (which is good). I've removed the numbering there entirely.